interface/ceed-fortran.c

```diff
@@ -479,7 +479,7 @@ void fCeedQFunctionCreateInterior(int* ceed, int* vlength,
 
   CeedQFunction *qf_ = &CeedQFunction_dict[CeedQFunction_count];
   *err = CeedQFunctionCreateInterior(Ceed_dict[*ceed], *vlength,
-                                     CeedQFunctionFortranStub,focca_c, qf_);
+                                     CeedQFunctionFortranStub, focca, qf_);
```

Comment:
Why? The string that comes directly from Fortran might not be NULL terminated.

Reply:
Wow, I have no idea how this happened, sorry for that.